requirements.txt

```diff
@@ -9,7 +9,7 @@ cloudpickle >= 2.0
 coolname >= 1.0.4
 croniter >= 1.0.12
 cryptography >= 36.0.1
-dateparser >= 1.0
+dateparser >= 1.1.2
```

Comment:
Why do we need the higher version?

Reply:
Tests were failing: https://github.com/PrefectHQ/prefect/actions/runs/3752658214/jobs/6375066682#step:12:236
I determined the cause to be: https://github.com/scrapinghub/dateparser/issues/1045 and the solution was implemented here: https://github.com/scrapinghub/dateparser/pull/1046.